calico-enterprise/operations/ebpf/install.mdx

```diff
@@ -156,6 +155,17 @@ default kernel used by Amazon Linux is recent enough to run eBPF mode, as is the
 18.04 image did not have a recent-enough kernel (but that may have changed by the time you read this).
 
 </TabItem>
+<TabItem label="MKE" value="MKE-5">
+
+`MKE` supports a number of base OSes; as long as the base OS chosen (such as Ubuntu 20.04 or RHEL 8.2) meets the kernel
```

Comment:
Not sure we should highlight 20.04, does MKE support a newer version that we could recommend?